src/drug_discovery/complex.py

```diff
@@ -143,25 +52,15 @@ def __post_init__(self):
         self.abfe = ABFE(parent=self)
         self.rbfe = RBFE(parent=self)
 
-        # Initialize the _hash
-        self._compute_hash()
-
-    def _compute_hash(self):
-        """Compute and update the hash based on protein and ligands"""
-        protein_hash = hash_file(self.protein.file_path)
-        ligands_hash = hash_strings([ligand.smiles for ligand in self.ligands])
-        self._hash = hash_strings([protein_hash, ligands_hash])
-
```

Comment:
i don't see the point of computing the hash anymore, because the constituent objects are backed by files